FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc

```diff
@@ -380,18 +331,18 @@ void CTPPSFastTrackingProducer::FastReco(int Direction,H_RecRPObject* station)
                     theta = CLHEP::pi - sqrt(thx*thx+thy*thy)*urad;
                     MatchCellId(cellId, recCellId_B, recTof_B, matchCellId, recTof); 
                 }
-                phi   = (Direction>0)?-atan2(thy,-thx):atan2(thy,thx); // defined according to the positive direction
-
+                //phi   = (Direction>0)?-atan2(thy,thx):atan2(thy,thx); // defined according to the positive direction
+                phi = atan2(thy,thx); // defined according to the positive direction
 
                 double px = partP*sin(theta)*cos(phi);
                 double py = partP*sin(theta)*sin(phi);
                 double pz = partP*cos(theta);
-                double  e = sqrt(partP*partP+ProtonMassSQ);
+                double  e = sqrt(partP*partP+pow(CLHEP::proton_mass_c2/GeV,2));
```

Comment:
better to left previous code

Reply:
So, should I keep the code with CLHEP? Should I change pow by a multiplication?

Reply:
@mundim , in the new PPSUtils there are const ProtonMassSQ, it should be used instead of pow.

Reply:
done.